frontend/src/components/applyForm/widgets/SelectWidget.tsx

```diff
@@ -43,35 +44,36 @@ function SelectWidget<
   S extends StrictRJSFSchema = RJSFSchema,
   F extends FormContextType = never,
 >({
-  schema,
   id,
-  options,
-  value,
-  required,
   disabled,
+  options,
   readonly,
-  multiple = false,
+  required,
+  schema,
+  value,
   autofocus = false,
-  updateOnInput = false,
+  multiple = false,
   rawErrors = [],
+  updateOnInput = false,
   // passing on* functions made optional
   onChange = () => ({}),
   onBlur = () => ({}),
   onFocus = () => ({}),
 }: UswdsWidgetProps<T, S, F>) {
+  const { title, description } = schema;
   // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
   const { enumOptions: opts, enumDisabled, emptyValue: optEmptyVal } = options;
-  const { title, description } = schema;
+  const enums = useMemo(() => (opts && opts?.length > 0 ? opts : []), [opts]);
```

Comment:
Use an empty array of select options if none are provided.

Reply:
[nit] don't think you need the optional chain since you're doing a null check for opts right before